release_notes/utils.py

```diff
@@ -272,3 +276,30 @@ def request_pull_requests_from_api(
                            f'{pending.keys()} is/are either not closed or cannot be found')
 
     return result
+
+
+def release_notes_files_into_docs(
+    repo_dir: str,
```

Comment:
you might default this to `.` / PWD - this would fit into how we built our github-actions..